mq.go

```diff
@@ -45,6 +50,8 @@ type MQ interface {
 	Error() <-chan error
 	// Close stop all consumers and producers and close connection to broker.
 	Close()
+	// Shows connection state
+	ConnectionState() int32
```

Comment:
To make it more readable for the client it makes sense to wrap int32 in custom type like `ConnectionState`. It allows us to add later sugar or completely change implementation without breaking backward compatibility.